docs/modules/ROOT/pages/how-to-guides/proc_release_application.adoc

```diff
@@ -66,94 +59,49 @@ metadata:
  namespace: dev-workspace <3>
 spec:
  application: demo-app <4>
- displayName: SRE Production <5>
- target:
-   namespace: managed-workspace <6>
+ data: <key> <5>
+ pipelineRef: <pipeline_ref> <6>
+ serviceAccount:  <service-account> <7>
+ target: managed-workspace <8>
 ----
 
 +
 <1> The name of the resource that you are creating.
 <2> The name of the release plan.
 <3> The development team's workspace.
 <4> The name of the application that you want to deploy to the managed workspace.
-<5> The <description>
-<6> The workspace to which the system deploys the application. This workspace is created by the Managed environment team (for example, your organization's SRE team)
-
-.*Verification*
-
-. Browse the application you are creating.
-. Select the *Overview* tab to visualize the Application workflow. If you have successfully added a Managed environment, the system displays it at the end of the Application workflow.
-
-== Creating a `ReleaseStrategy` object
-
-.*Prerequisites*
+<5> An unstructured key used for providing data for the release Pipeline.
+<6> An optional reference to a Pipeline that would be executed before the release Pipeline.
+<7> The name of the service account to use in the Pipeline to gain elevated privileges.
```

Comment:
May be worth saying this is also optional and only used if a pipeline is provided in 6